styles/modules/modals/_editListing.scss

```diff
@@ -60,6 +60,18 @@
           }
         }
       }
+
+      .tagsSection {
+        .select2 {
+          margin-bottom: $padSm;
+        }
+      }
+
+      .categoriesSection {
+        .select2 {
+          margin-bottom: $padSm;
+        }
```

Comment:
If you make the changes to the select2 tags above, the hrs after the control will be positioned right, and you won't need the extra padding here.